port/unix/omrsysinfo.c

```diff
@@ -3408,24 +3442,23 @@ readCgroupFile(struct OMRPortLibrary *portLibrary, int pid, OMRCgroupEntry **cgr
 		}
 		cursor = subsystems;
 		do {
-			const char *cgName = NULL;
+			int32_t i = 0;
 
 			separator = strchr(cursor, ',');
 			if (NULL != separator) {
 				*separator = '\0';
 			}
 
-			cgName = getCgroupNameForSubsystem(portLibrary, cgEntryList, cursor);
-
-			if (NULL == cgName) {
-				rc = addCgroupEntry(portLibrary, &cgEntryList, hierId, cursor, cgroup);
-				if (0 != rc) {
-					goto _end;
+			for (i = 0; i < sizeof(supportedSubsystems) / sizeof(supportedSubsystems[0]); i++) {
+				if (J9_ARE_NO_BITS_SET(available, supportedSubsystems[i].flag)
```

Comment:
This should use `OMR_ARE_NO_BITS_SET` instead of the `J9` version.  This will solve a compile break in this change.